prov/util/src/util_shm.c

```diff
@@ -166,8 +195,13 @@ int smr_create(const struct fi_provider *prov, struct smr_map *map,
 
 	fd = shm_open(attr->name, O_RDWR | O_CREAT | O_EXCL, S_IRUSR | S_IWUSR);
 	if (fd < 0) {
```

Comment:
I'd be explicit and check for `EEXIST` and fail immediately for all other cases.

Reply:
Good call. Thanks!